src/mock-doc/element.ts

```diff
@@ -297,6 +314,25 @@ export class MockSVGElement extends MockElement {
   getTotalLength(): number {
     return 0;
   }
+  getComputedTextLength(): number {
+    return 0;
+  }
+  createSVGPoint() {
+    return {
+      matrixTransform: defaultSVGMatrix,
+      x: 0,
+      y: 0,
+    };
+  }
+  getBBox() {
+    return { x: 0, y: 0, width: 10, height: 10 };
+  }
+  getCTM() {
+    return defaultSVGMatrix;
+  }
+  getScreenCTM() {
+    return defaultSVGMatrix;
```

Comment:
I think the return value of `getScreenCTM` needs to be a [DOMMatrix](https://developer.mozilla.org/en-US/docs/Web/API/DOMMatrix). Although it's similar to an `SVGMatrix`, it looks like there are different properties/methods on the two